schema/base/landCover.yaml

```diff
@@ -0,0 +1,83 @@
+---
+"$schema": https://json-schema.org/draft/2020-12/schema
+title: Land Cover Schema
+description: representation of the Earth's natural surfaces 
+type: object
+properties:
+  id:
+    "$ref": ../defs.yaml#/$defs/propertyDefinitions/id
+  geometry:
+    description: classification of surface, can correspond to the 'natural' tag from OpenStreetMap
+    unevaluatedProperties: false
+    oneOf:
+      - "$ref": https://geojson.org/schema/Polygon.json
+      - "$ref": https://geojson.org/schema/MultiPolygon.json
+  properties:
+    unevaluatedProperties: false
+    allOf:
+      - "$ref": ../defs.yaml#/$defs/propertyContainers/overtureFeaturePropertiesContainer
+      - "$ref": ../defs.yaml#/$defs/propertyContainers/levelContainer
+    required:
+      - subType
+      - minZoom
+      - maxZoom
+    properties:
+      subType:
+        description: type of surface represented
+        type: string
+        enum:
+          - barren
+          - crop
+          - forest
+          - grass
+          - mangrove
+          - moss
+          - shrub
+          - snow
+          - urban
+          - wetland
+      minZoom:
+        description: recommended minimum zoom according to bing tile zoom levels
```

Comment:
i've added [resolution](https://github.com/OvertureMaps/schema/pull/118/files#diff-93a22edf02ed0f205d46d9f18da43e699ec780037852e240283dd126d413fab1R49) as an optional column - resolution feels weird by itself since they are not round units. Zoom level, imo, feels pretty standard for anyone building maps. 

since the proposed data set is multi-resolution, recommended zooms + resolution gives the user all the necessary information

Reply:
yeah i think this works well